keystone_api/plugins/slurm.py

```diff
@@ -108,13 +109,13 @@ def get_cluster_usage(account_name: str, cluster_name: str, in_hours: bool = Tru
 
     cmd = split(f"sshare -nP -A {account_name} -M {cluster_name} --format=GrpTRESRaw")
 
-    usage = re.findall(r'billing=(.*),fs', subprocess_call(cmd))[0]
-
-    if not usage.isnumeric():
+    try:
+        usage = re.findall(r'billing=(.*),fs', subprocess_call(cmd))[0]
```

Comment:
This can technically be empty, so I'm handling that here. It might be better to just error out here instead of assuming 0.